libs/design-system/web-component-card/src/lib/web-component-card.component.ts

```diff
@@ -0,0 +1,22 @@
+import { CommonModule } from '@angular/common';
+import { ChangeDetectionStrategy, Component, input } from '@angular/core';
+import { ButtonModule } from '@hra-ui/design-system/button';
+
```

Comment:
Please add a doc comment to the component. Doc coverage will likely fail otherwise